packages/r/reproc/xmake.lua

```diff
@@ -0,0 +1,24 @@
+package("reproc")
+
+    set_description("a cross-platform C/C++ library that simplifies starting, stopping and communicating with external programs.")
+    set_homepage("https://github.com/DaanDeMeyer/reproc")
+    set_license("MIT")
+
+    add_urls("https://github.com/DaanDeMeyer/reproc.git")
+    add_versions("v14.2.4", "8aea3aef64c37a7ed0db759a7128a80c2540c808")
+
+    add_deps("cmake")
+
+    add_configs("c++", { description = "Build reproc C++ library.", default = true, type = "boolean" })
+
+    on_install(function(package)
+        local configs = {}
+        table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "RelWithDebInfo"))
+        table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))
+        table.insert(configs, "-DREPROC++=" .. (package:config("c++") and "ON" or "OFF"))
```

Comment:
`++=`? does cmake support it?

Reply:
Apparently so: 

https://github.com/DaanDeMeyer/reproc#user

Reply:
ok

Reply:
I've added pthread and ws2_32 syslinks.